### DISCUSSION TITLE
Ebooks: have we considered adding readarr to swizzin?

### DISCUSSION THREAD

**Discussion by fredguile** [swizzin/swizzin#1078] [Feature Requests]
Hey team, I'm not sure I'm finding anything for handling ebooks... have we considered adding readarr into swizzin?

**Comment by liaralabs** [comment]
You mean this readarr? :smile: 

https://github.com/swizzin/swizzin/blob/master/scripts/install/readarr.sh

**Reply by fredguile** [reply]
Yeah thanks @liaralabs !
